src/oneTrust/endpoints/getAssessment.ts

```diff
@@ -0,0 +1,678 @@
+/* eslint-disable max-lines */
+import * as t from 'io-ts';
+
+/**
+ * reference: https://developer.onetrust.com/onetrust/reference/exportassessmentusingget
+ * Return types of the "Get Assessments" endpoint
+ */
+export const OneTrustApprover = t.type({
+  /** ID of the user assigned as an approver. */
+  id: t.string,
+  /** ID of the workflow stage */
+  workflowStageId: t.string,
+  /** Name of the user assigned as an approver. */
+  name: t.string,
+  /** More details about the approver */
+  approver: t.type({
+    /** ID of the user assigned as an approver. */
+    id: t.string,
+    /** Full name of the user assigned as an approver. */
+    fullName: t.string,
+    /** Email of the user assigned as an approver. */
+    email: t.union([t.string, t.null]),
+    /** Whether the user assigned as an approver was deleted. */
+    deleted: t.boolean,
+    /** The assignee type */
+    assigneeType: t.union([t.string, t.null]),
+  }),
+  /** Assessment approval status. */
+  approvalState: t.union([
+    t.literal('OPEN'),
+    t.literal('APPROVED'),
+    t.literal('REJECTED'),
+  ]),
+  /** Date and time at which the assessment was approved. */
+  approvedOn: t.union([t.string, t.null]),
+  /** ID of the assessment result. */
+  resultId: t.union([t.string, t.null]),
+  /** Name of the assessment result. */
+  resultName: t.union([
+    t.literal('Approved - Remediation required'),
+    t.literal('Approved'),
+    t.literal('Rejected'),
+    t.literal('Assessment suspended - On Hold'),
+    t.string,
+    t.null,
+  ]),
+  /** Name key of the assessment result. */
+  resultNameKey: t.union([t.string, t.null]),
+});
+
+/** Type override */
+export type OneTrustApprover = t.TypeOf<typeof OneTrustApprover>;
+
+export const OneTrustAssessmentStatus = t.union([
```

Comment:
nit for next time: would define this as an enum then use `valuesOf(EnumName)` to create the codec.